privaterelay/templates/profile.html

```diff
@@ -67,226 +72,229 @@
-                  <span class="relay-email-label email-forwarding-label card-small-text">{% ftlmsg 'profile-label-forward-emails' %}</span>
-                  <span class="text-dark demi primary-email-address card-small-text">{{ request.user.email }}</span>
-                </section>
-              </div>
+              
+            </div> 
 
+            <div class="c-alias-copy">
+              <button title="Click to copy"
+                data-id="{{ relay_address.id }}"
+                data-clipboard-text="{{ relay_address.address }}@{{ RELAY_DOMAIN }}"
+                class="relay-address click-copy ff-Met xflx jst-cntr al-cntr">
+                  {{ domain_address.address }}@{% user_email_domain user_profile %}
+                  <span class="alias-copied-icon"></span>
+                  <span class="alias-copied-message">{% ftlmsg 'profile-label-copied' %}</span>
+              </button>
+            </div>
 
-              <!-- delete email address -->
-              <section class="relay-email-column column-delete center-value">
-                <form action="{% url 'emails-index' %}" method="POST" class="delete-email-form flx jst-cntr al-cntr" data-delete-relay="{{ domain_address.address }}@{% user_email_domain user_profile %}">
-                  <input type="hidden" name="method_override" value="DELETE">
-                  <input type="hidden" name="api_token" value="{{ user_profile.api_token }}">
-                  <input type="hidden" name="domain_address_id" value="{{ domain_address.id }}">
-                  <button type="submit" name="delete" value="" class="dashboard-delete-button delete-relay-email-button" aria-label="Delete this alias">
-                    <span class="remove-label relay-email-label card-small-text">{% ftlmsg 'profile-label-delete' %}</span>
-                  </button>
-                </form>
-              </section> <!--end delete-->
+            <div class="c-alias-additional-info js-alias-details">
+                <div class="c-alias-data-collection">
+                  <!-- forwarding message -->
+                  <div class="c-alias-data forward ">
+                    <img class="" src="/static/images/icon-email.svg" alt="">
+                    <div>
+                      <span class="c-alias-data-label">{% ftlmsg 'profile-label-forward-emails' %}</label>
+                      <span class="c-alias-data-value">{{ request.user.email }}</span>
+                    </div>
+                  </div>
+
+                  <!-- created by date -->
+                  <div class="c-alias-data created-by">
+                    <img class="" src="/static/images/icon-calendar-gray.svg" alt="">
+                    <div>
+                      <span class="c-alias-data-label">{% ftlmsg 'profile-label-created' %}</label>
+                      <span class="c-alias-data-value">{{ relay_address.created_at|date:"M j, Y" }}</span>
```

Comment:
Not sure if we can localize the date formats to work internationally.

Reply:
Providing a pattern here is not localizable, you'll need to use something that will generate the pattern for you correctly. For example, see the Fluent DATETIME function, which lets you choose the components you need, and it will generate the localized pattern.

https://projectfluent.org/fluent/guide/functions.html

Reply:
In JS-land this is the [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) API.

Reply:
Issue filed https://github.com/mozilla/fx-private-relay/issues/907